mod/wikirate_source/set/type/source/source.rb

```diff
@@ -88,12 +85,30 @@ def has_file_or_text?
   
 end
 
+def is_url? url
```

Comment:
Use def with parentheses when there are parameters.<br>Rename `is_url?` to `url?`.